### PR TITLE
Adds a cream pie smite.

### PR DESCRIPTION
**Pull request by BlackMajor**
Sometimes you just want to lightly slap someone for being shit instead of causing damage to them.

Also reorganises the smite menu list from least to most dangerous.

### CODE REVIEW

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1294,6 +1294,9 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 			if(!puzzle_imprison(target))
 				to_chat(usr,"<span class='warning'>Imprisonment failed!</span>")
 				return
+		if(ADMIN_PUNISHMENT_PIE)
+			var/obj/item/reagent_containers/food/snacks/pie/cream/nostun/creamy = new /obj/item/reagent_containers/food/snacks/pie/cream/nostun(get_turf(target))
```
Comment (Poojawa) [inline code/modules/admin/verbs/randomverbs.dm]: > nostun

Rejected.

Comment (Ghommie) [reply code/modules/admin/verbs/randomverbs.dm]: What the above fellow said.

Comment (Useroth) [reply code/modules/admin/verbs/randomverbs.dm]: +1

Comment (Useroth) [inline code/modules/admin/verbs/randomverbs.dm]: also, you don't need the full path for `new`, if you've declared it in the variable
```suggestion
			var/obj/item/reagent_containers/food/snacks/pie/cream/nostun/creamy = new(get_turf(target))
```

Comment (BlackMajor) [reply code/modules/admin/verbs/randomverbs.dm]: Thanks fam

**Review by silicons — APPROVED** [review]

**Pull request merged**